src/__spec_helper__/test-utils.js

```diff
@@ -192,8 +194,16 @@ const backgroundProps = [
   ["backgroundRepeat", "background-repeat", "no-repeat"],
 ];
 
-const getDefaultValue = (value) => {
-  if (typeof value === "number") {
+export const getDefaultValue = (value) => {
+  const spaceArrayLength = space.length - 1;
+  const parsedValue = +value;
+  if (typeof value === "string" && value > spaceArrayLength) {
```

Comment:
This only makes sense if it is a string containing a number, if it's something like "auto" this doesn't work.

Reply:
This function should be something like: 

```js
const getDefaultValue = (value) => {
  if (typeof value === "number" && value < space.length) {
    return space[value];
  }
  return value;
};
```

We don't number numerical values above the length of the space array, and any string should just be set as a string

Reply:
@samtjo I think it actually does work correctly, because if value is `auto` then it will return false

Reply:
@samtjo The style-system works in such a way that if I give for example m={1} it will take element 2 from the array. But if you put m=“1” it will also take the 2nd element from the space array.
At least that’s how it behaves when I checked in browser.

Reply:
Yeah you're right, I didn't realise styled-system worked with numerical string like that, in that case I think it's fine 🙂